buildenv/jenkins/triggerGrinder.groovy

```diff
@@ -1,46 +1,49 @@
 #!groovy
-
-def json_path = args.size() ? args[0] : "disabledTestParser/output.json."
-def json = readJSON file: json_path
-def jenkins_url = "https://ci.adoptopenjdk.net/job/Grinder/buildWithParameters"
-
+node {
+   checkout scm
+  //  : [$class: 'GitSCM',
+  //                           branches: [[name: "${scm.branches[0].name}"]],
+  //                           extensions: [
+  //                               [$class: 'CleanBeforeCheckout'],
+  //                               [$class: 'CloneOption', reference: ref_cache],
+  //                               [$class: 'RelativeTargetDirectory', relativeTargetDir: 'aqa-tests']],
+  //                           userRemoteConfigs: [[url: "${gitConfig.getUrl()}"]]
+                        // ]
+  stage('readJSON')
+  {
+    def json_path = "${WORKSPACE}/aqa-tests/disabledTestParser/output.json"
+    def json = readJSON file: json_path
+    /**
+    Get each of the key-value pairs for each json value.
+    If the issue for a job is closed, run grinder on it.
+    Otherwise ignore it */
+    json.eachWithIndex { dict, _ ->
+      if (dict["GIT_ISSUE_STATUS"] == "closed") {
+        run_grinder(dict)
+      }
+    }
+  }
+}
 /**
- This runs when we have found a job w/ git_issue_status = closed.
- Collects all parameters and runs grinder
- */
+  This runs when we have found a job w/ git_issue_status = closed.
+  Collects all parameters and runs grinder
+  */
 def run_grinder(map) {
-  def list = []
-  def queryString
+  stage("running grinder")
+  {
+    def childParams = []
+    def queryString
 
-  map.each { k, v ->
-    list.add(k + "=" + v)
-  }
+    map.each { k, v ->
+      if (k!="ISSUE_TRACKER_STATUS" && k!="ISSUE_TRACKER") {
+          childParams << string(name: "${k}", value: "${v}")
+      }
+    }
 
-  queryString = list.join("&")
-  url = jenkins_url + "/?" + queryString
+    for (e in childparams) {
+      println e
+    }
 
-  // The Grinder trigger will look something like this,
-  // but it requires tokens which I do not have yet.
-  // def url = new URL(jenkins_url + "/?" + queryString)
-  // def connection = url.openConnection()
-  // connection.with {
-  //     doOutput = true
-  //     requestMethod = 'POST'
-  //     outputStream.withWriter { writer ->
-  //         writer << queryString
-  //     }
-  //     println content.text
-  // }
-
-  println url
-}
-
-/**
- Get each of the key-value pairs for each json value.
- If the issue for a job is closed, run grinder on it.
- Otherwise ignore it */
-json.eachWithIndex { dict, _ ->
-  if (dict["GIT_ISSUE_STATUS"] == "closed") {
-    run_grinder(dict)
+    build job: Grinder, parameters: childParams, propagate: false
```

Comment:
This line sets up and triggers a Grinder job to be run.

We need extra information to verify that this Grinder actually got triggered.  @amir-waheed - Can we print out the information relating to it, so we can more easily find it to see how it behaved.